db/migrate/20200512214951_create_shelter_review.rb

```diff
@@ -0,0 +1,11 @@
+class CreateShelterReview < ActiveRecord::Migration[5.1]
+  def change
+    create_table :shelter_reviews do |t|
+      t.string :title
+      t.string :rating
+      t.string :content
+      t.string :image
+      t.timestamp 
```

Comment:
Meg mentioned that it is good practice to include timestamps and for this class especially I thought this was true.

Reply:
Definitely! I include timestamps with almost every migration. One thing, it works with singular timestamp?

Reply:
uhoh maybe it doesn't. DANG THOSE PLURALS!